sign-in.txt

```diff
@@ -1,5 +1,7 @@
 ﻿This is an incredibly useful text document.
 //TODO: Be sure you have created a feature branch from develop 
 //TODO: Please put your name and github username AS SHOWN below ... 
-//       ... then commit, pull origin develop, merge develop into your branch, push origin feature, and review your pull request
-Brian Bansenauer : Bansenauer-Cascadia
+  ... then commit, pull origin develop, merge develop into your branch, push origin feature, and review your pull request
+  
+- Brian Bansenauer, github:Bansenauer-Cascadia
+- Keith Rutler, github:keithrutler
```

Comment:
Sim,
It seems like you have pulled in the develop branch correctly, but not yet added you name to the list. 
There are several other names, beside Keith added now, so you will need to  go through the PULL-Merge-Commit-PUSH cycle once more. 
Hang in there - the more practice the better.
Also be sure to review someone else's code and leave a comment; respond to a comment on your code too.

Reply:
I think I may have it this time....

Reply:
You got it Sim - congrats.
Not required, but if you want to practice the process again you could update your entry so it matches the format,
yourname, github: yourgithubname